weaviate/embedded.py

```diff
@@ -29,8 +29,8 @@
 class EmbeddedOptions:
     persistence_data_path: str = os.environ.get("XDG_DATA_HOME", DEFAULT_PERSISTENCE_DATA_PATH)
     binary_path: str = os.environ.get("XDG_CACHE_HOME", DEFAULT_BINARY_PATH)
-    version: str = "1.19.12"
-    port: int = 6666
+    version: str = "latest"
+    port: int = 8080
```

Comment:
might be a problem if it is the default weaviate port by default. Should we rather do 8079?

Reply:
Hum. I thought about 8080 as it's the suggested port for the docker compose config tool.

Reply:
Hi duda, sorry forgot about this one! I think we should make it easy to have embedded running next to a local weaviate version - if we use 8079 as default then they won't conflict